backend/union/policy/eplno.go

```diff
@@ -91,7 +100,7 @@ func (p *EpLno) CreateEntries(entries ...upstream.Entry) ([]upstream.Entry, erro
 	if err != nil {
 		return nil, err
 	}
-	e, err := p.lnoEntries(entries)
+	e, err := p.lnoEntries(context.Background(), entries)
```

Comment:
Not sure if this way of instantiating a new context is appropiate in this context, this is my first time doing a bit more of Go and a background context seemed to me the most sensible for this use case, but I actually have no idea 😅.

Reply:
You either use `context.Background()` if this is really a background operation, or `context.TODO()` to say you'd really like to have a context here but you don't have one, or you go context plumbing and plumb in ctx into this function and all its callers etc.

If you look at the calls to the About backend method in the upstream.go file you'll see it is called from the background so assuming this code gets moved into upstream.go then it should be `context.Background()`